.github/CONTRIBUTING.md

```
@@ -13,23 +13,60 @@ Side refers to the Sipe Design System, our primary UI component library aimed at
 
 ### Fork and Clone the Repository
 First, fork the [repository](https://github.com/sipe-team/side) and then clone it locally:
-```sh
+```bash
 git clone https://github.com/your-username/side.git
 ```
 
-### Install Dependencies
-Install the necessary dependencies:
-```sh
+### Setup Toolchain with mise
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)